### ISSUE TITLE
Would be nice if data could by given in ember data format?

### ISSUE THREAD

**Issue by vvsalmin** [sir-dunxalot/ember-google-charts#12]
Wouldn't it be much easier if you could do charts easily out of store.query without needing format data as array of arrays.. I mean if data could be given in format

`
[
  {type: "work": amount: 12}, {...}
]
`

**Comment by sir-dunxalot** [comment]
Yep, great idea. I believe something like #10 would fix this, though we would have to support multiple chart types. Can you confirm this is the kind of thing you are thinking of?

**Comment by sir-dunxalot** [comment]
This is a change I would like to make but am not going to get around to it anytime soon.

If anyone wants to submit a PR with tests I'd love to make this happen.
